### ISSUE TITLE
Bundle is not finding `nim` on Windows

### ISSUE THREAD

**Issue by RickBarretto** [arturo-lang/arturo#1637]
**Describe the bug**
I tried to test the new `bundle` feature, but this seems to be not finding the `nim` compiler on Windows.
My `nim` is into my system's `PATH`, and I can call it from the Powershell or Bash.

**To Reproduce**
![image](https://github.com/arturo-lang/arturo/assets/78623871/86c06ff5-fd70-4f40-b7e4-d0f25b106e5e)

**Expected behavior**
Find nim as it is for Unix systems.

**Desktop (please complete the following information):**
 - OS: Win11
 - Version v/0.9.83 b/3112 @ 2024-05-19T21:22:08-03:00

**Comment by drkameleon** [comment]
Hmm... 🤔 

Can you please try one thing: if you go to **vm/bundle/generator.nim** and change the entire content of `commandExists` to:

```nim
if findExe(cmd) == "":
        echo "\t`" & cmd & "` command required; cannot proceed!"
        quit(1)
```

does it progress past this step?

**Comment by drkameleon** [comment]
Also, tiny hint on the `--bundle` syntax:

- This works with executable scripts or packages
- The general idea is to point it to *one* specific script file, e.g. `arturo --bundle somescript.art` (this will create a `somebundle` binary)
- You may also point it to a package folder, but the `info.art` *must* declare an `executable:` in that case (which specific script is to be the entry point for the executable)
- Also, the `--as:` option is only used if you want to rename the target binary to sth different than the default one ^, e.g. `arturo --bundle --as:myscript somescript.art` (that is: instead of creating a `somescript` binary, rename it to `myscript` 😉 )

**Comment by drkameleon** [comment]
> * Fixed on [Fix bundler issues on Windows #1636](https://github.com/arturo-lang/arturo/pull/1636)

Totally forgot about the open issue!

Great! ;)
